### DISCUSSION TITLE
Search Query Generation Prompt

### DISCUSSION THREAD

**Discussion by ItsHotdogFred** [open-webui/open-webui#4777] [General]
# Bug Report

## Installation Method

I have used docker to install open web ui for OpenAI

## Environment

- **Open WebUI Version:** 0.3.13 (Latest)

- **Operating System:** Hosted on NixOS
- **Browser (if applicable):**  I'm using firefox

**Confirmation:**

- Yes I have read and followed all the instructions provided in the README.md.
- Yes I am on the latest version of both Open WebUI and Ollama.
- Yes I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Should generate a web query response to then search up based on the users question

## Actual Behavior:

Just enters in exactly what the user said and not using the prompt shown in the Interface tab.

## Description

When using the web search feature it seems like it's not using the web prompt shown in the interface tab. Instead it just searches up exactly what the user said. I also asked someone in the discord server who also seemed to have the same problem. He 

**Bug Summary:**
The web search tool is just using the users response as the search query 

## Reproduction Details

**Steps to Reproduce:**
Use the web search function... That's is pretty much it lol

## Logs and Screenshots
![image](https://github.com/user-attachments/assets/9af815de-cd03-4e6e-b6ac-7c3ab5913968)

The person I asked on discord who also had the same problem posted  some logs which prove that the title generation works but not the search query generations

[message.txt](https://github.com/user-attachments/files/16683693/message.txt)

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

I can give more information if needed

**Comment by CaptainMoney** [comment]
It looks like nobody answered you so I will Mr. Hotdog. 
I put this in the Search Query Prompt for Interface in Admin Panel:

```
Given the user's message and interaction history, decide if a web search is necessary. You must be concise and exclusively provide a search query if one is necessary. Refrain from verbose responses or any additional commentary. Prefer suggesting a search if uncertain to provide comprehensive or updated information. If a search isn't needed at all, respond with an empty string. Default to a search query when in doubt. Today's date is {{CURRENT_DATE}}.

User Message:
{{prompt:end:4000}}
```
Not sure why this works but it works wayyy better than that prompt that comes pre populated.
Got it from here: https://github.com/open-webui/open-webui/compare/v0.3.21...main

![image](https://github.com/user-attachments/assets/b8e55a16-db43-4967-a849-af44f26ba5b0)
![image](https://github.com/user-attachments/assets/84e72385-25db-4a73-a76a-864d9ae32693)

**Reply by DiazOlveraCo** [reply]
is it just me or they removed this search query prompt edit option

**Reply by dynm** [reply]
admin settings -> Interface
